docs/DeveloperGuide.md

```diff
@@ -479,6 +478,32 @@ Use case ends
  
     Use case resumes from step 4.
 
+**Use case (UC4): Search for reservation availability**
+
+**MSS**
+
+1. User requests to search for reservation availability at a date and time.
+2. RHRH shows the availability for the indicated date time.  
+   Use case ends.
+
+**Extensions:**
+
+- 1a. User requests to search for reservation availability at a date.
+    - 1a1. RHRH shows all the available slots on the indicated date.  
+      Use case ends.
+
+- 1b. User requests to search for reservation availability at a time.
+    - 1b1. RHRH shows the availability for the indicated time today.  
+      Use case ends.
+
+- 1c. RHRH detects information missing from the command entered.
+    - 1c1. RHRH requests user to enter the command again with sufficient information.  
```

Comment:
Hi Thang, you can refer to  the google docs to see the format for the use case please follow what me and Chetwin wrote.
For this you should have another line 1c2 `user enters command again` or `user enters missing info`. then another line that days `Steps 1c1 - 1c2 are repeated until the command is correctly entered` and then another line that days `use case resumes from 1 /1a /1b` depending on what the user entered.

Reply:
Ok sorry, I fixed it.